src/Tizen.NUI.Components/Controls/Navigation/AppBar.cs

```diff
@@ -215,12 +253,60 @@ public View TitleContent
             }
         }
 
+        /// <summary>
+        /// Action views of AppBar.
+        /// Action views are added to ActionContent of AppBar.
+        /// The Action and ActionButton styles of AppBarStyle are applied to actions only by setting Actions.
+        /// If you do not want to apply Action and ActionButton styles to action views, then please use ActionContent.Add() instead.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public IEnumerable<View> Actions
+        {
+            get
+            {
+                return appBarActionViews;
+            }
+            set
+            {
+                if (appBarActionViews != null)
+                {
+                    foreach (var oldAction in appBarActionViews)
+                    {
+                        if (ActionContent.Children?.Contains(oldAction) == true)
+                        {
+                            ActionContent.Children.Remove(oldAction);
+                        }
+                    }
+                }
+
+                appBarActionViews = value;
+
+                if (appBarActionViews == null)
+                {
+                    return;
+                }
+
+                foreach (var action in appBarActionViews)
+                {
+                    // Apply Action and ActionButton styles.
+                    if ((action is Button) && (appBarStyle?.ActionButton != null))
+                    {
+                        ((Button)action)?.ApplyStyle(appBarStyle.ActionButton);
+                    }
+                    else if (appBarStyle?.ActionView != null)
+                    {
+                        action?.ApplyStyle(appBarStyle.ActionView);
+                    }
+
+                    ActionContent.Add(action);
```

Comment:
ActionContent could be a null
```suggestion
                    ActionContent?.Add(action);
```

Reply:
Thank you for your comment.
I have updated code to do null check ActionContent in the beginning as you suggested below.